ouroboros-consensus-cardano/tools/db-analyser/Block/Alonzo.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE DeriveAnyClass             #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE NamedFieldPuns             #-}
+{-# LANGUAGE OverloadedStrings          #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Block.Alonzo () where
+
+-- * Prelude
```

Comment:
The `StyleGuide.md` calls for this grouping, with no comments. (I _think_ Plutus gets its own -- hasn't really come up before that I recall.)

```
import           Control.Applicative
import           Data.Aeson (FromJSON (..), (.:), (.:?))
import qualified Data.Aeson as Aeson
import           Data.Aeson.Types (FromJSONKey (..))
import qualified Data.Map.Strict as Map
import           Data.Text (Text)

import           Plutus.V1.Ledger.Api (defaultCostModelParams)

import qualified Cardano.Ledger.Alonzo.Genesis as Alonzo
import qualified Cardano.Ledger.Alonzo.Language as Alonzo
import qualified Cardano.Ledger.Alonzo.Scripts as Alonzo
import qualified Cardano.Ledger.BaseTypes as Ledger
```

Reply:
I hope to get this done https://github.com/input-output-hk/cardano-ledger-specs/issues/2452 (if ledger folks give me 👍 ) but till now I want to keep this code 1:1 with what `cardano-node` is having. How it should look I would love to address on the PR that will implement https://github.com/input-output-hk/cardano-ledger-specs/issues/2452. Does it make sense?

Reply:
👍